sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/ConcurrencyControl.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.models;
+
+import com.azure.storage.blob.BlobClient;
+
+/**
+ * Defines values for ConcurrencyControl.
+ * {@link #NONE}
+ * {@link #E_TAG}
+ * {@link #VERSION_ID}
+ */
+public enum ConcurrencyControl {
```

Comment:
I don't love the name concurrency control. I think it's technically correct, but it sounds to me more like it's intended to manage like number of threads or something. And etags are for optimistic concurrency, but I don't think versions fits in that linguistic space. That might just be me. It's more about like ensuring a consistent view of the blob for the whole download. It's also weird because they behave differently. Etag will throw if it changes, but version id will never throw as long as the version is not deleted. 
... I don't have any suggestions here though.

Reply:
ConsistencyControl?

Reply:
Yea I think that works better in my mind. Good one.

Reply:
`ConsistentReadControl` ?

Reply:
I like that as well

Reply:
Going with Kamil's suggestion

Reply:
Resolved